storage/postgres/types.go

```diff
@@ -293,3 +293,20 @@ func (v *Visibility) FromDTO(visibility *types.Visibility) {
 		v.PlatformID.Valid = true
 	}
 }
+
+func getJSONText(item json.RawMessage) sqlxtypes.JSONText {
+	if len(item) == len("null") && string(item) == "null" {
```

Comment:
can't the `len(item) == len("null")` check be omitted?

Reply:
If the `item` is very large this will prevent it from an unnecessary unmarshalling.